### PR TITLE
✨ Add commands for German language

### PR DESCRIPTION
**Pull request by JanB1**
Added some commands and definitions for the German language.

### CODE REVIEW

**Comment by francois-rozet** [comment]
Hello @JanB1, thank you for your contribution! I will review it shortly.

**Review by francois-rozet — CHANGES_REQUESTED** [review]
Looks good to me overall. I left two comments for you to fix, and we can merge afterwards!

**Comment by JanB1** [comment]
I implemented the requested changes.

File: packages/sleek.sty [hunk 1]
```diff
@@ -127,6 +127,11 @@
     backrefpages = {pages}
 }
 
+\DefineBibliographyStrings{german}{
+    backrefpage = {page},
+    backrefpages = {pages}
```
[the remark this answers is not in the document]
Comment (JanB1) [reply packages/sleek.sty]: Yeah, didn't notice that one, good catch!

**Comment by francois-rozet** [comment]
I just realized that the `babel` package already translates the captions and table of contents. So I dropped the related code in `sleek.sty` but kept the translations in `sleek-theorems.sty`. Thanks again for the contribution and sorry for the little mess 😅

**Pull request merged**